app/controllers/api/v1/decks_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Api::V1::DecksController < Api::V1Controller
+  before_action :restrict_access
```

Comment:
This will validate the user's session and set up the `current_user` if there is one.